examples/CAII-CBS/bCAII-LiHa.gwl

```diff
@@ -1,173 +0,0 @@
-A;Water;;Trough 100ml;2;;400.000000;;;2

```

Comment:
Any idea why these examples are being deleted?

Reply:
Since .xlsx and .gwl files are in the .gitignore file I thought I had mistakenly added them. But clearly they are supposed to be a part of the repo. I'll add them back.